fiftyone/utils/eta.py

```
@@ -43,6 +44,77 @@
     etal.VideoSemanticSegmenter,
 )
 
+_tf2_patched = False
+
+
+def _patch_tf2_detection_model():
```

Ruff (0.14.8)

Missing return type annotation for private function `_patch_tf2_detection_model`

Add return type annotation: `None`

(ANN202)